src/CoreAnimation/.gitignore

```diff
@@ -1 +0,0 @@
-/*.g.cs
```

Comment:
Remove the entire file?

Reply:
yup, there's shouldn't be an .gitignore there (and that's covered in parent dirs)
it could have been done a different PR but I found it while auditing every files do nullability